### ISSUE TITLE
new installer type: none

### ISSUE THREAD

**Issue by NikkyAI** [NikkyAI/alpacka#23]
allow installing the pack and running all command without actually doing anything special, like installing mc versions or forge versions

**Comment by copygirl** [comment]
That would be a simple git clone, right. I don't think this is needed..?

**Comment by NikkyAI** [comment]
yes it would be.. and i just to have it to be able to do any commands that work with targets without requiring to have multimc installed or making a server

like `alpacka init bare ...`

**Comment by copygirl** [comment]
I guess you can have an install bare but... wouldn't - or rather **shouldn't** `init bare <url>` do exactly the same as `git clone <url>`?

**Comment by NikkyAI** [comment]
init creates a new modpack.. you cannot give it a url
